.relay/config.yml is a relative path (#10498)
getsentry/sentry-docs: docs/product/relay/options.mdx

sidebar_order: 3
---

> Configuration for Relay is recorded either in the file `.relay/config.yml` (relative to working directory), or when noted, can be overridden through an environment variable. To change the config location, pass the `--config` option to any Relay command:

Reviewer: ```suggestion
Configuration for Relay is recorded either in the file `.relay/config.yml` (relative to the working directory), or when noted, can be overridden through an environment variable. To change the config location, pass the `--config` option to any Relay command:
```

Is that better?

> Configuration for Relay is recorded either in the file `.relay/config.yml` (relative to the working directory), or when noted, can be overridden through an environment variable. To change the config location, pass the `--config` option to any Relay command:

Reviewer 2: ```suggestion
Configuration for Relay is typically recorded in the file `.relay/config.yml` (relative to the working directory). Configuration options can be overridden through environment variables. To change the config location, pass the `--config` option to any Relay command:
```
